src/Common/src/Internal/Cryptography/CryptoThrowHelper.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal static class CryptoThrowHelper
```

Comment:
Could you put this under Interop\Windows somewhere?  It's definitely Windows-specific, and I think it would be good to have that included in the path.